### ISSUE TITLE
Midterm Peer Review - kc594

### ISSUE THREAD

**Issue by kc594** [tinali0923/Volatility-Prediction#11]
The goal of this project is to use historical volatility in combination with additional information about the stock such as stock fundamentals to predict a more accurate future volatility.

It was smart to initially look into the actual values that your features take on, and consider the scale of them in relation to each other.  While normalization is crucial for some models, I have had experience where, depending on the model, normalization may not help, so it would be good to be open to try with and without normalized data, and make sure to understand your models well to know when it is necessary or not.  If you try models where normalization is not crucial it may help the interpretability of weights/importance of features.

The feature vs volatility graphs are very interesting to see, as individual relations between  feature and the outcome could provide you with new insights moving forward.  A brief explanation of what the features are that you are using and adding would be helpful for someone reviewing the project.  Although the individual feature relations to the outcome could be useful, it is possible that certain combinations of the features would be much more significant, so finding a way to visualize this may provide you with important ideas in the rest of your work, but the combination of close price and a certain feature distinguished by color is a great start and very informative.

The analysis of the graphs is also very useful and shows you have put some thought into your creation and understanding of the results.  The coloring of the closing price makes it easy to see that there are some yellow points around mostly blue, for example, and this leads me to think there may be some outliers in the data, so it could be interesting to experiment with removing outliers from the data to try to find stronger relations.

Possibly keeping features with more than 10% could be useful, as this does not seem like an unreasonable amount to interpolate with the mean/mode/median.  Experimenting with different thresholds of the missing data or different methods of interpolation could actually give you better results than you would expect.  I would advise to be very open to different experiments before throwing out the data.

The time span of the initial graphs in your report under the Description section appear to span many years, so it could be interesting to try different length spans of time in your experiments, as you may find more significant results with a shorter (or longer) time span.  You have a great opportunity to try a time series model, and you could try building a model with only hourly/daily/yearly data to see how the results change.

Don't be afraid to try different sizes training and validation sets, giving yourself more training data might be useful for building the model since you have it available - 80/20 or 90/10 even could be good splits to try and see how your model changes.

I agree that the relationships may be non-linear, and the linear regression gave you a good starting point to explore from here.  You have a lot of good ideas I am excited to see what models you find to be good fits and the results you obtain.  Good luck!
